devai-cli/src/devai/constants.py

```diff
@@ -0,0 +1,7 @@
+"""Constants used throughout the application."""
```

Comment:
can we delete this `constants.py` file?
 - commands are using 
https://github.com/GoogleCloudPlatform/genai-for-developers/blob/main/devai-cli/src/devai/commands/constants.py#L2